config.go

```diff
@@ -0,0 +1,37 @@
+package viewproxy
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type configRouteEntry struct {
+	Url               string   `json:"url"`
+	LayoutFragmentUrl string   `json:"layout"`
+	FragmentUrls      []string `json:"fragments"`
+}
+
+func readConfigFile(filePath string) ([]configRouteEntry, error) {
+	file, err := os.Open(filePath)
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Go: Who needs a monad when you have `return`?? 😆 

But seriously, it is pretty easy to visually parse the early returns to see the potential error cases